packages/main/test/specs/Dialog.spec.js

```diff
@@ -80,6 +80,36 @@ describe("Dialog general interaction", () => {
 
 		browser.keys("esc");
 	});
+
+	it.only("resizable", () => {
```

Comment:
```it("resizable", () => {```

Currently this is the only test that is being run from the whole page. After removing the keyword ```only```, all the tests would be run. As much as I can see, the resizable dialog is initially opened. This might be a problem for the other tests. The best solution would be if all the dialogs are initially closed and in the test you open the needed dialog.

Reply:
Sorry, I forgot to remove this `it.only`. I also removed the setTimeout on Dialog.html to restore the other tests as well.